packages/utils/src/document-upload.ts

```diff
@@ -0,0 +1,76 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { DocumentReference } from "@medplum/fhirtypes";
+import * as fsPromises from "fs/promises";
+// import * as fs from "fs";
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { sizeInBytes } from "@metriport/core/util/string";
+import axios from "axios";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiUrl = getEnvVarOrFail("API_URL");
+const patientId = getEnvVarOrFail("PATIENT_ID");
+
+const metriportApi = new MetriportMedicalApi(apiKey, {
+  baseAddress: apiUrl,
+});
+
+async function readFileAsync(filePath: string) {
+  try {
+    return await fsPromises.readFile(filePath, "utf8");
+  } catch (err) {
+    console.error(err);
+  }
+}
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function put(url: string, data: any) {
+  const response = await axios.put(url, data, {
+    headers: {
+      "Content-Length": sizeInBytes(data),
+    },
+  });
+
+  return response.data;
+}
+
+async function main() {
+  const docRef: Partial<DocumentReference> = {
+    description: "Second degree ankle burn treatment",
+    type: {
+      text: "Burn management Hospital Progress note",
+      coding: [
+        {
+          code: "100556-0",
+          system: "http://loinc.org",
+        },
+      ],
+    },
+    context: {
+      period: {
+        start: "2023-10-10T14:14:17Z",
+      },
+      facilityType: {
+        text: "My Clinic Name - Acute Care",
+      },
+    },
+  };
+  const fileContent = await readFileAsync("./src/pdf_example.pdf");
+  // const fileContent = fs.readFileSync("./src/pdf_example.pdf");
```

Comment:
left-over?

Reply:
i left these intentionally, thinking both versions could be useful for different file types (one works for xml, but doesnt work for pdf. second works for pdf, and i didnt test it for xml)